internal/lib/stats/metrics.go

```diff
@@ -0,0 +1,368 @@
+				Help:      "Number of throttled period intervals.",
+				LabelKeys: baseLabelKeys,
+			}, {
+				Name:      "container_cpu_cfs_throttled_seconds_total",
+				Help:      "Total time duration the container has been throttled.",
+				LabelKeys: baseLabelKeys,
+			},
+		},
+		"cpuLoad": {
+			{
+				Name:      "container_cpu_load_average_10s",
+				Help:      "Value of container cpu load average over the last 10 seconds.",
+				LabelKeys: baseLabelKeys,
+			}, {
+				Name:      "container_tasks_state",
+				Help:      "Number of tasks in given state",
+				LabelKeys: append(baseLabelKeys, "state"),
+			},
+		},
+		"disk": {
+			{
+				Name:      "container_fs_inodes_free",
+				Help:      "Number of available Inodes",
+				LabelKeys: append(baseLabelKeys, "device"),
+			}, {
+				Name:      "container_fs_inodes_total",
+				Help:      "Number of Inodes",
+				LabelKeys: append(baseLabelKeys, "device"),
+			}, {
+				Name:      "container_fs_limit_bytes",
+				Help:      "Number of bytes that can be consumed by the container on this filesystem.",
+				LabelKeys: append(baseLabelKeys, "device"),
+			}, {
+				Name:      "container_fs_usage_bytes",
+				Help:      "Number of bytes that are consumed by the container on this filesystem.",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+		},
+		"diskio": {
+			{
+				Name:      "container_fs_reads_bytes_total",
+				Help:      "Cumulative count of bytes read",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_reads_total",
+				Help:      "Cumulative count of reads completed",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_sector_reads_total",
+				Help:      "Cumulative count of sector reads completed",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_reads_merged_total",
+				Help:      "Cumulative count of reads merged",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_read_seconds_total",
+				Help:      "Cumulative count of seconds spent reading",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_writes_bytes_total",
+				Help:      "Cumulative count of bytes written",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_writes_total",
+				Help:      "Cumulative count of writes completed",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_sector_writes_total",
+				Help:      "Cumulative count of sector writes completed",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_writes_merged_total",
+				Help:      "Cumulative count of writes merged",
+				LabelKeys: append(baseLabelKeys, "device"),
+			},
+			{
+				Name:      "container_fs_write_seconds_total",
+				Help:      "Cumulative count of seconds spent writing",
```

Comment:
```suggestion
				Help:      "Cumulative count of seconds spent writing.",
```

Reply:
naturally there's no consistency in cadvisor on whether to end with a period: https://github.com/google/cadvisor/blob/master/metrics/prometheus.go#L674 I think we should keep these consistent with the cadvisor equivalent for now, to allow easier comparison